src/app2.js

```diff
@@ -0,0 +1,52 @@
+const express = require('express');
+const mongoose = require('mongoose');
```

Comment:
Also, can we separate the server code from the `src` folder?

Something like:

```
.
|-- public/
|-- src/
|-- server/
|-- package.json
|-- ...
```

Need to deploy the server separately